samples/rabbitmq/cluster/Rabbit_8/ClusteredRabbitMQ.sln.DotSettings

```diff
@@ -0,0 +1,6 @@
+<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
don't commit the DotSettings file

Reply:
Hmm, the other samples seems to have them? https://github.com/Particular/docs.particular.net/blob/master/samples/rabbitmq/simple/Rabbit_7/SimpleRabbitMQ.sln.DotSettings

Reply:
Removed now